OrbitGl/PickingManager.h

```diff
@@ -101,22 +97,24 @@ class PickingManager {
   void Pick(uint32_t a_Id, int a_X, int a_Y);
   void Release();
   void Drag(int a_X, int a_Y);
-  Pickable* GetPicked();
-  Pickable* GetPickableFromId(uint32_t id);
-  bool IsDragging() const;
+  std::weak_ptr<Pickable> GetPicked();
```

Comment:
I know you just changed the return type, but maybe it's a good occasion to add const here?

Reply:
Done